src/io/leopard/toLeopard.ts

```diff
@@ -276,24 +266,25 @@ export default function toLeopard(
 
       // If the block contains a variable or list dropdown,
       // get the code to grab that variable now for convenience
-      let varName: string = null;
       let selectedVarSource: string = null;
       let selectedWatcherSource: string = null;
+      let varInputId: string = null;
+      let isSpriteVar: boolean = null;
       if ("VARIABLE" in block.inputs) {
-        varName = block.inputs.VARIABLE.value.toString();
+        varInputId = (block.inputs.VARIABLE.value as { id: string }).id;
```

Comment:
The type assertion here seems to be because the block's `VARIABLE` input can only be narrowed to `BlockInput.Any`. Something to look into in a future PR?

Reply:
Yep, marked this on your issue #92. Thanks!